src/ln/router.rs

```diff
@@ -35,30 +35,31 @@ pub struct Route {
 	pub hops: Vec<RouteHop>,
 }
 
-struct DirectionalChannelInfo {
-	src_node_id: PublicKey,
-	last_update: u32,
-	enabled: bool,
-	cltv_expiry_delta: u16,
-	htlc_minimum_msat: u64,
-	fee_base_msat: u32,
-	fee_proportional_millionths: u32,
+//Here, DirectionalChannelInfo, ChannelInfo, NodeInfo and NetworkMap are pub(crate) only for log_* macros access
+pub(crate) struct DirectionalChannelInfo {
```

Comment:
Hmm, it seems weird to put the logger in log macros and then expose all these things, maybe just implement Display for these types directly instead of going through the indirection?